README.md

```diff
@@ -1103,3 +1103,14 @@ $config = [
 ];
 
 ```
+
+
+## Migrate mobile app extensions from old to new
+
+This helper change all mobile app extensions from old version to the new one
+
+```
+php /var/www/html/freepbx/rest/lib/appMigrationHelper.php
```

Comment:
Are you sure SCL should not be enabled?

Reply:
yes, I used freepbx_db.conf that only loads db and not functions, and avoided the use of PHP function array_columns()